gpt_engineer/steps.py

```diff
@@ -19,12 +21,24 @@ def setup_sys_prompt(dbs):
 Step = TypeVar("Step", bound=Callable[[AI, DBs], List[dict]])
 
 
+def get_prompt(dbs):
+    """While we migrate we have this fallback getter"""
+    assert (
+        "prompt" in dbs.input or "main_prompt" in dbs.input
+    ), "Please put your prompt in the file `prompt` in the project directory"
+
+    if "prompt" not in dbs.input:
+        print(
+            colored("Please put the prompt in the file `prompt`, not `main_prompt", "red")
+        )
```

Comment:
Maybe something like this is a bit more readable but not a big deal (especially since this is just for the transition)

```python
def get_prompt(dbs):
    if ("main_prompt" in dbs.input) or ("prompt" not in dbs.input):
        colored("Please put your prompt in a file called `prompt` in the project directory", "red")
        
```